src/redisai.c

```diff
@@ -11,8 +11,9 @@
 
 #include "rmutil/alloc.h"
 #include "util/arr_rm_alloc.h"
+#include "util/dict.h"
+#include "util/dict.c"
```

Comment:
Why are we including `dict.c`?

Reply:
sorry. setted it just to experiment on dict.c and then I forgot to properly adjust cmakefile. Done